modules/external-roles/README.md

```diff
@@ -0,0 +1,32 @@
+This module is for creating specific iam roles in specific projects for existing service account.
+
+Module accept a list of service accounts, projects and roles in next data structure passed to roles_map variable:
```

Comment:
The module accepts..... and roles. The following data structure is passed to roles_map variable: